tests/test_annotypes.py

```diff
@@ -162,6 +162,12 @@ def another_func(self):
         assert list(Sub1.call_types) == ["name"]
         assert list(Sub2.call_types) == ["name"]
 
+        # check subclassing, in particular that Any can be passed for
+        #  subclass checking
+        assert issubclass(Sub2, Root)
```

Comment:
Not a fan of overriding a builtin. Can we make move the code from Malcolm that uses issubclass to a new classmethod Anno.matches_type(inst)?

Reply:
I've added CallTypesMeta.matches_type and modified Malcolm Model to use it.